### PR TITLE
Feature/todo badge

### PR DESCRIPTION
**Pull request by nysthee**
This is how it looks at the moment. There was only one trending project which had a todo-file so I couldn't test any more. I must be noted that I'm not a great designer but I think it clarifies he purpose of this feature.

screenshot from 2015-04-22 21 06 07

### CODE REVIEW

**Comment by mcls** [comment]
Cool, I'll review this later this week. Maybe @stelianfirez can design a better badge.

**Comment by stelianfirez** [comment]
I can make a couple of alternatives.

File: app/views/projects/show.html.erb [hunk 1]
```diff
@@ -2,7 +2,12 @@
 <% content_for(:meta_description) { [@project_details.full_name, @project_details.project.description].compact.join(' - ') } %>
 <section class="project_details">
   <% cache ['project_header', @project_details.cache_key] do %>
-    <h2><%= link_to @project_details.username, "https://github.com/#{@project_details.username}" , target: '_blank'%> / <%= link_to @project_details.name, "https://github.com/#{@project_details.username}/#{@project_details.name}", target: '_blank' %></h2>
+    <h2>
+      <%= link_to @project_details.username, "https://github.com/#{@project_details.username}" , target: '_blank'%> / <%= link_to @project_details.name, "https://github.com/#{@project_details.username}/#{@project_details.name}", target: '_blank' %>
+      <% if @project_details.has_todo? %>
+        <%= image_tag('badge_silver.png', alt: 'Todo Badge', size: '67x68') %>
+        <% end %>
```
Comment (jeroenj) [inline app/views/projects/show.html.erb]: Check your indentation here ;-)

File: app/models/project/download.rb [hunk 2]
```diff
@@ -43,6 +43,7 @@ def rescue_reopen_error
   # @param file [String]
   def unzip_to_source_files(file)
     Zip::File.open_buffer(file) do |zip_file|
+      if zip_file.glob('**/.rubocop_todo.yml').present? then @project.has_todo = true end
```
Comment (jeroenj) [inline app/models/project/download.rb]: You can rewrite this as `@project.has_todo = zip_file.glob('**/.rubocop_todo.yml').present?`.

**Comment by stelianfirez** [comment]
refactorcop_todo_01
refactorcop_todo_02
refactorcop_todo_03
refactorcop_todo_04

I made a version with the "tools" icon that @nysthee used and one with the "checkmark" that's popular in todo apps. Both with two backgrounds. If you guys think that another pictogram might be more appropriate in our case, I can give that one a try too.

**Comment by mcls** [comment]
@stelianfirez I vote for number 2 (grey checkmark).

**Comment by nysthee** [comment]
@stelianfirez I second @mcls's opinion

**Comment by stelianfirez** [comment]
What would be the best way for me to add the css and svg?

**Comment by nysthee** [comment]
I can add you ass a collaborator on my fork? Or is there a better option @mcls?

**Comment by mcls** [comment]
Thanks @nysthee and @jeroenj. I'll merge this, then Stelian can commit the logo later.

**Comment by stelianfirez** [comment]
so should I make a branch then?

**Comment by mcls** [comment]
sure, I'm on skype btw

**Pull request merged**